src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,15 +1,37 @@
 package core.mate.academy.model;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Add some custom fields that could be only in Excavator
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+
+    private int size;
+
     public Excavator() {
     }
 
+    public Excavator(int size) {
+        this.size = size;
+    }
+
     @Override
     public void doWork() {
         System.out.println("Excavator started to work");
     }
+
+    @Override
+    public List<? extends Machine> get() {
```

Comment:
This method should not be here because this class should not implement interface "MachineProducer". Create class ExcavatorProducer for it